bundle/edu.gemini.ags/src/main/scala/edu/gemini/ags/impl/SingleProbeStrategy.scala

```diff
@@ -199,7 +198,7 @@ case class SingleProbeStrategy(key: AgsStrategyKey, params: SingleProbeStrategyP
       base <- ctx.getBaseCoordinates.asScalaOpt.toList
       so <- candidates
       pa = SingleProbeStrategy.calculatePositionAngle(base.toNewModel, so)
-      ctxSo = ctx.withPositionAngle(pa.toOldModel)
```

Comment:
I think that that's something that should be fairly easy to do, and that we should aim towards.